dist/Dockerfile.e2e/Dockefile

```diff
@@ -0,0 +1,14 @@
+FROM centos:7
+
+ENV HOME="/root"
+
+RUN mkdir -p "${HOME}/cincinnati"
+WORKDIR ${HOME}/cincinnati
+
+# Get oc CLI
+RUN mkdir -p ${HOME}/bin && \
+    curl https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz 2>/dev/null | tar xzf - -C ${HOME}/bin/ oc
```

Comment:
Unlikely to matter (who would put shell-sensitive chars in `HOME`?), but would be nice to quote `"${HOME}/bin"` here and in the line above.

Reply:
Added quotes (to WORKDIR as well)